src/components/card/card.jsx

```diff
@@ -11,16 +10,20 @@ var Card = React.createClass({
     e.stopPropagation();
     this.props.onActionsClick.call(this, this.props);
   },
-  prerenderImage: function() {
-    return React.createElement('img', {
-      src: Card.DEFAULT_THUMBNAIL
-    });
+  onImageError: function() {
+    var imageEl = this.refs.imageEl.getDOMNode();
+    imageEl.src = Card.DEFAULT_THUMBNAIL;
+    imageEl.onerror = null;
+  },
+  componentDidMount: function () {
```

Comment:
i guess we could create the image element here too. do you think that's better? i can't think of another way to do it